src/SchemaForm.js

```diff
@@ -84,11 +92,11 @@ class SchemaForm extends React.Component {
             mapper = _.merge(this.mapper, this.props.mapper);
         }
         let forms = merged.map(function(form, index) {
-            return this.builder(form, this.props.model, index, mapper);
+            return this.builder(form, this.props.model, index, mapper, this.onChange, this.builder);
```

Comment:
why do we need to pass `this.builder` as an argument now, but didn't before?

Reply:
not quite sure, but I think that it's a one of the things that I've seen in obsolete versions of library, so moved it here, but I'll check it role again and maybe remove it